src/Mailgun/Resource/Api/Suppressions/Complaint/IndexResponse.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * Copyright (C) 2013-2016 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+namespace Mailgun\Resource\Api\Suppressions\Complaint;
+
+use Mailgun\Resource\Api\PaginationResponse;
+use Mailgun\Resource\ApiResponse;
+
+/**
+ * @author Sean Johnson <sean@mailgun.com>
+ */
+class IndexResponse implements ApiResponse
+{
+    use PaginationResponse;
+
+    /**
+     * @var Complaint[]
+     */
+    private $items;
+
+    /**
+     * @param Complaint[] $items
+     * @param array       $paging
+     */
+    public function __construct(array $items, array $paging)
+    {
+        $this->items = $items;
+        $this->paging = $paging;
+    }
+
+    /**
+     * @param array $data
+     *
+     * @return Complaint[]
```

Comment:
Like my previous comments, this is a named constructor that is calling `new self()` so this annotation should be updated to `IndexResponse`. Can you do it please? Thanks,